README.md

```diff
@@ -908,6 +909,23 @@ Here's a list of other literature covering Bash:
 * ["Bourne-Again SHell manual"](https://www.gnu.org/software/bash/manual/) in many formats, including HTML, Info, TeX, PDF, and Texinfo.  Hosted at <https://www.gnu.org/>.  As of 2016/01, this covers version 4.3, last updated 2015/02/02.
 * [Bash 3.2 Man page](https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man1/bash.1.html) hosted at Apple's Mac Developer Library site.  As of 2016/01, this covers version 3.2, last updated 2006/09/28.
 
+# Other resources
+
+* [awesome-bash](https://github.com/awesome-lists/awesome-bash) is a curated list of Bash scripts and resources
+
+* [awesome-shell](https://github.com/alebcay/awesome-shell) is another curated list of shell resources
+
+* [bash-it](https://github.com/Bash-it/bash-it) provides a solid framework for using, developing and maintaining shell scripts and custom commands for your daily work.
+
+* [learnyoubash](https://github.com/denysdovhan/learnyoubash) helps you write your first bash script
+
+* [liquidprompt](https://github.com/nojhan/liquidprompt) is a full-featured prompt for both bash and zsh
```

Comment:
this item seems like common dotfiles, like many other. I [have dotfiles too](https://github.com/denysdovhan/dotfiles). Unfortunately, this seems like promoting of that project which is proprietary solution.

Instead of that, I suggest to replace this link by [GitHub Dotfiles](http://dotfiles.github.io/) link. This link will be a good starting point.